zio-http/src/test/scala/zhttp/internal/HttpRunnableSpec.scala

```diff
@@ -76,7 +76,7 @@ abstract class HttpRunnableSpec extends ZIOSpecDefault { self =>
         id       <- Http.fromZIO(DynamicServer.deploy(app))
         url      <- Http.fromZIO(DynamicServer.wsURL)
         response <- Http.fromFunctionZIO[SocketApp[HttpEnv]] { app =>
-          ZIO.scoped {
+          ZIO.scoped[HttpEnv] {
```

Comment:
Scala 3 makes a mess here without this.
I haven't investigated why. If someone has time please find the root cause of this as there might be something wrong with the implementation of `scoped`.